datalist-polyfill.js

```diff
@@ -2,387 +2,389 @@
-					}
-					$dataListSelect.setAttribute( 'aria-hidden', ( !visible ).toString() );
-					
-					// bind the keyup event on the related datalists input
-					switch( eventType ) {
-						case 'focus':
-							$eventTarget.addEventListener( 'keyup', inputInputList );
-							
-							$eventTarget.addEventListener( 'focusOut', changesInputList, true );
-						break;
-						case 'blur':
-							$eventTarget.removeEventListener( 'keyup', inputInputList );
-							
-							$eventTarget.removeEventListener( 'focusOut', changesInputList, true );
-						break;
-					}
-				}
-			}
-		};
-
-		// function regarding changes to the datalist polyfilling created select
-		var changeDataListSelect = function( event ) {
-
-			var $eventTarget = event.target,
-				eventTargetTagName = $eventTarget.tagName.toLowerCase();
-
-			// check for whether the events target was a select or an option
-			if ( eventTargetTagName && ( eventTargetTagName === 'select' || eventTargetTagName === 'option' ) ) {
-
-				var $select = ( eventTargetTagName === 'select' ) ? $eventTarget : $eventTarget.parentNode,
-					$datalist = $select.parentNode,
-					message = $datalist.title,
-					eventType = event.type,
-					// ENTER and ESC
-					visible = ( eventType === 'keyup' && ( event.keyCode !== keyENTER && event.keyCode !== keyESC ) );
-				
-				// change or mouseup event or ENTER key
-				if ( eventType === 'change' || eventType === 'click' || ( eventType === 'keyup' && event.keyCode === keyENTER ) ) {
-
-					var list = $datalist.id,
-						$inputList = document.querySelector( 'input[list="' + list + '"]' ),
-						selectValue = $select.value;
-
-					// input the selects value into the input on a change within the list
-					if ( $inputList !== null && typeof( selectValue ) !== 'undefined' && selectValue.length > 0 && selectValue !== message ) {
-						var inputListValue = $inputList.value,
-							lastSeperator,
-							multipleEmails = ( $inputList.type === 'email' && $inputList.multiple );
-						
-						// in case of type=email and multiple attribute, we need to set up the resulting inputs value differently
-						if ( multipleEmails && ( lastSeperator = inputListValue.lastIndexOf( ',' ) ) > -1 ) {
-							$inputList.value = inputListValue.slice( 0, lastSeperator ) + ',' + selectValue;
-						} else {
-							$inputList.value = selectValue;
-						}
-						
-						// set the visibility to false afterwards, as we're done here
-						visible = false;
-					}
-				}
-
-				// toggle the visibility of the select according to previous checks
-				if ( visible ) {
-					$select.removeAttribute( 'hidden' );
-				} else {
-					$select.setAttributeNode( document.createAttribute( 'hidden' ) );
-				}
-				$select.setAttribute( 'aria-hidden', ( !visible ).toString() );
-			}
-		};
-
-		// binding the focus event - matching the input[list]s happens in the function afterwards
-		document.addEventListener( 'focus', changesInputList, true );
-
-
-	}
+(function() {
```

Comment:
Ohh, the re-indent. As a random bystander, I'd probably recommend making large formatting changes in separate PRs from logic changes - then, code review can happen on logic changes, but autoformat PRs can go through pull-request without too much scrutiny since they shouldn't change any logic :smile: